src/wazuh_mcp_server/utils/platform_compat.py

```
@@ -0,0 +1,59 @@
+"""
+Platform-specific compatibility handling for Pydantic V1/V2.
+Addresses Fedora-specific issues while maintaining macOS/Ubuntu compatibility.
+"""
+
+import os
+import platform
+import sys
```

Ruff (0.11.9)

`sys` imported but unused

Remove unused import: `sys`

(F401)